### PR TITLE
[WIP. Not Ready] Refactor JsonRpcWallet

### PR DESCRIPTION
**Pull request by kalambet**
1. JsonRpc Wallet refactored into the Signer and TxExecutor
2. InProcessTxEx is removed.

### CODE REVIEW

**Review by robGG1997 — APPROVED** [review]

File: src/ChainSafe.Gaming/Web3/Evm/JsonRpc/JsonRpcTransactionExecutor.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+            if (transaction.GasLimit == null)
+            {
+                var feeData = await provider.GetFeeData();
+                transaction.MaxFeePerGas = new HexBigInteger(feeData.MaxFeePerGas);
+            }
+
+            var rpcTxParams = transaction.ToRPCParam();
+            return await provider.Perform<string>("eth_sendTransaction", rpcTxParams);
+        }
```
Comment (oleksandrchainsafe) [inline src/ChainSafe.Gaming/Web3/Evm/JsonRpc/JsonRpcTransactionExecutor.cs:54]: Shouldn't we sign the transaction at some point?

**Review by oleksandrchainsafe — CHANGES_REQUESTED** [review]
All good. The only issue is transaction signing.

File: src/ChainSafe.Gaming/Web3/Evm/JsonRpc/JsonRpcTransactionExecutor.cs (hunk 1 above)
Comment (kalambet) [reply src/ChainSafe.Gaming/Web3/Evm/JsonRpc/JsonRpcTransactionExecutor.cs:54]: Yes, my bad, working on it 🥹

**Comment by kalambet** [comment]
Too much changes in the codebase. Closing this one and start from the scratch.

**Pull request closed without merging**